examples/textual_inversion/textual_inversion_mlflow.py

```diff
@@ -437,12 +437,6 @@ def parse_args():
         default=10,
         help="Log every X updates steps. Default to 10.",
     )
-    parser.add_argument(
-        "--seed",
```

Comment:
Removing `--seed` argument will break existing testbench model configurations. I suggest keeping this.

Reply:
I remove it because I accidentally add it in the previous PR. The original code already provides that argument. cc: @yeomni